hbase-client/src/main/java/org/apache/hadoop/hbase/RegionMetrics.java

```diff
@@ -141,9 +141,9 @@ default String getNameAsString() {
   Size getUncompressedStoreFileSize();
 
   /**
-   * @return the data locality of region in the regionserver.
+   * @return the data at local weight of this region in the regionserver
    */
-  float getDataLocality();
+  long getBlocksLocalWeight();
```

Comment:
RegionMetrics is IA.Public, if we are intending this patch to go to branch-2 and branch-1, we should better not modify existing methods, only add new ones.